plugins/inspect/config_files_inspector.rb

```diff
@@ -43,7 +43,7 @@ def packages_with_config_files
     # use leading slash to decide between lines containing package names
     # and lines containing config files
     chunks = output.split("\n").slice_before { |l| !l.start_with?("/") }
-    chunks.reject { |pkg, *cfiles| cfiles.empty? }.map(&:first)
+    chunks.reject { |pkg, *cfiles| cfiles.empty? }.map(&:first).uniq
```

Comment:
Unused block argument - `pkg`. If it's necessary, use `_` or `_pkg` as an argument name to indicate that it won't be used.